src/Zoxive.HttpLoadTesting.Client/Domain/GraphStats/Services/GraphStatsService.cs

```diff
@@ -68,6 +68,52 @@ FROM HttpStatusResult
 
             return result;
         }
+
+        public async Task<IEnumerable<StatusCodeStatDto>> GetStatusCodes(Filters filters)
+        {
+            if (!filters.Period.HasValue)
+                throw new ArgumentNullException("Filter.Period must have a value");
+            if (!filters.Frequency.HasValue)
+                throw new ArgumentNullException("Filter.Frequence must have a value");
+
+            var minuteMilliseconds = Math.Round(filters.Period.Value * 60000);
+
+            var frequency = filters.Frequency;
+
+            var httpStatusWhere = _httpStatusRepository.CreateWhereClause(filters, out var sqlParams);
+
+            var min = await _connection.QueryFirstAsync<long>($@"
+SELECT
+MIN(RequestStartTick) as Min
+FROM HttpStatusResult
+{httpStatusWhere}
```

Comment:
i was thinking about this on the drive home. I think we need to remove this here, and in the other Func that does this same MIN() query